Revise Test Team Rep duties and qualifications (#183)
ironprogrammer/test-handbook: team-reps/index.md

Test Team Rep duties include:

- Write bi-weekly [Test Team Update](https://make.wordpress.org/updates/tag/test/) posts and publish them on the [Team Updates](https://make.wordpress.org/updates/) page.
- Write monthly [Month in Test](https://make.wordpress.org/test/category/month-in-test/) posts and publish them on [Make WordPress Test](https://make.wordpress.org/test/).

> - Write agendas for bi-weekly `<test-chat>` sessions ([example](https://make.wordpress.org/test/tag/test-chat-agenda/)).

Reviewer: ```suggestion
- Write agendas for bi-weekly `<test-chat>` sessions ([examples](https://make.wordpress.org/test/tag/test-chat-agenda/)).
```
Pluralize because link moves to tag archive.

> - Run alternating weekly `<test-chat>` ([example](https://wordpress.slack.com/archives/C03B0H5J0/p1626181220122300)) and `<Patch Testing>` ([example](https://wordpress.slack.com/archives/C03B0H5J0/p1785423614352719)) sessions in [#core-test](https://wordpress.slack.com/messages/core-test/).

Reviewer: Don't need to use Copilot's wording here, but I do agree that maintaining links to these guide pages is helpful.

> - Run alternating weekly `<test-chat>` ([example](https://wordpress.slack.com/archives/C03B0H5J0/p1626181220122300)) and `<Patch Testing>` ([example](https://wordpress.slack.com/archives/C03B0H5J0/p1785423614352719)) sessions in [#core-test](https://wordpress.slack.com/messages/core-test/).

Reviewer: If the header used at the start of each session in Slack has changed, then use whatever the current convention is. It does look inconsistent.

- Write `<test-chat>` session recaps and publish them on [Make WordPress Test](https://make.wordpress.org/test/).

> - Run Voice Chat sessions in [#core-test](https://wordpress.slack.com/archives/C03B0H5J0) every **1st Thursday of the month**.

Reviewer: ```suggestion
- Run Voice Chat sessions in [#core-test](https://wordpress.slack.com/archives/C03B0H5J0) on the first Thursday of every month.
```
Just a nitpick. Use of strong text here isn't consistent with other guide conventions.

- Help new contributors with their WordPress testing-related queries.
- Help raise awareness of testing needs, especially for upcoming WordPress releases.
- Raise issues or red flags that other teams should be aware of or discussing.

> These duties are shared between the two Reps (see **Rep Responsibilities** on [the Team Rep page](https://make.wordpress.org/updates/team-reps/#team-rep-orientation)).

Reviewer: ```suggestion
These duties are shared between all team Reps (see **Rep Responsibilities** on [the Team Rep page](https://make.wordpress.org/updates/team-reps/#team-rep-orientation)).
```
There wasn't always two reps, and afaik there isn't anything requiring exactly two, so this small change should accommodate future team shapes ("all" is optional).
